user/src/com/google/gwt/user/tools/templates/maven/pom.xmlsrc

```diff
@@ -83,9 +83,9 @@
           <moduleName>@moduleName</moduleName>
           <moduleShortName>@moduleShortName</moduleShortName>
           <failOnError>true</failOnError>
-          <!-- GWT compiler 2.8 and above requires 1.8, hence define sourceLevel here if you use
+          <!-- GWT compiler 2.12.0 and above requires 11, hence define sourceLevel here if you use
                a different source language for java compilation -->
-          <sourceLevel>1.8</sourceLevel>
+          <release>11</release>
```

Comment:
Sorry, that wasn't clear - the `sourceLevel` tag hasnt changed here, but javac and maven-compiler-plugin now support just a <release> tag instead of both <source> and <target>. The <release> tag and --release arg has a slightly different meaning too, it ensures you don't use newer JRE classes/members even if building with a newer JRE.

https://maven.apache.org/plugins/maven-compiler-plugin/examples/set-compiler-release.html